doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst

```diff
@@ -0,0 +1,154 @@
+* Added fast switching between radio states for the nRF54H20 SoC.
+* Added fast radio channel switching for the nRF54H20 SoC.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following Bluetooth samples:
+
+  * :ref:`direct_test_mode` sample
+
+    * Added experimental support for the HCI interface.
+    * Changed internal sample API.
+
+Enhanced ShockBurst samples
+---------------------------
+
+* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following ESB samples:
+
+  * :ref:`esb_prx` sample
+  * :ref:`esb_ptx` sample
+
+Multicore samples
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following peripheral samples:
+
+  * :ref:`multicore_hello_world` sample
+
+Peripheral samples
+------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following peripheral samples:
+
+  * :ref:`radio_test` sample
+
+* Updated:
+
+  * The CLI command ``fem tx_power_control <tx_power_control>`` replaces ``fem tx_gain <tx_gain>`` .
+    This change applies to the sample built with the :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
+
+Zephyr
+======
+
+..
+  ##update commits
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``0051731a41fa2c9057f360dc9b819e47b2484be5``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline 0051731a41 ^23cf38934c
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^0051731a41
+
+The current |NCS| main branch is based on revision ``0051731a41`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Zephyr samples
+--------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` in the following Zephyr samples:
+
+  * :zephyr:code-sample:`blinky`
+  * :ref:`zephyr:hello_world`
+  * :zephyr:code-sample:`fade-led`
+
```

Comment:
@grochu @KAGA164 Shouldn't we also add a bluetooth observer?

Reply:
I think it's not in scope for NCS 2.6.99-cs2